cockatrice/src/settingscache.cpp

```diff
@@ -89,6 +91,12 @@ void SettingsCache::setCardInfoViewMode(const int _viewMode) {
     settings->setValue("cards/cardinfoviewmode", cardInfoViewMode);
 }
 
+void SettingsCache::setHighlightWords(const QString _highlightWords) {
+    // Words are seperated by a comma and you can not use spaces in words
+    highlightWords = _highlightWords.simplified().replace(" ", "").split(",");
```

Comment:
I originally had it in the conversion in the `chatview.cpp` file, but it seemed clunky as I want highlightWords to return a QStringList, not a QString